contracts/sol6/mock/Token.sol

```diff
@@ -220,8 +220,9 @@ contract Token is StandardToken {
     }
 
     // save some gas by making only one contract call
-    function burnFrom(address _from, uint256 _value) public returns (bool) {
-        transferFrom(_from, msg.sender, _value);
+    function burnFrom(address _from, uint256 _value) public
+        returns (bool) {
+        assert( transferFrom( _from, msg.sender, _value ) );
```

Comment:
Remove the spaces in the assert statement, ie. `assert(transferFrom( _from, msg.sender, _value ));`